cmd/server-main.go

```diff
@@ -341,6 +341,7 @@ func serverHandleCmdArgs(ctxt serverCtxt) {
 	globalLocalNodeName = GetLocalPeer(globalEndpoints, globalMinioHost, globalMinioPort)
 	nodeNameSum := sha256.Sum256([]byte(globalLocalNodeName))
 	globalLocalNodeNameHex = hex.EncodeToString(nodeNameSum[:])
+	globalLocalPoolIdx = getLocalNodePoolIdx(globalLocalNodeName)
```

Comment:
I would suggest that this is returned via the `GetLocalPeer()` function itself to avoid doing double the work.

Reply:
I am sending a correct fix.

Reply:
also a single server can belong to two pools at the same time.

Reply:
I remove this global now, we don't need it in this form anyway.